### DISCUSSION TITLE
Thanks for yabai

### DISCUSSION THREAD

**Discussion by bas-ie** [asmvik/yabai#896] [General]
Open source development is often rather thankless, so I wanted to make a point of saying: thanks for yabai and skhd! They are saving my ass right now, compelled to work on a Macbook in a new job and really missing my i3. When a Big Sur update broke hotkeys/scripting addition, sure enough I waited a few days and your new patch fixed everything.

I don't have the resources for a long-term sub at the moment, but have made a one-off donation via Patreon. Thank you so much for your work @koekeishiya (and other contributors).

**Comment by asmvik** [comment]
Thank you for the support and kind words!
